### ISSUE TITLE
Update crop area help text

### ISSUE THREAD

**Issue by jkeck** [projectblacklight/spotlight#1670]
After #1498 is merged the help text is not exactly accurate.

From @ggeisler:

```
Adjust the image so that the rectangle contains the area you want to use as the #{thing}. Click "Save changes" to save the cropped area.
```
Where `thing` is:
- site thumbnail or site masthead (in the exhibit's appearances settings page)
- thumbnail or masthead (in the case of a browse categories or pages)
- contact photo

**Comment by jkeck** [comment]
@ggeisler I noticed a potential wrinkle here.  There is one area that this doesn't quite account for, which is the actual Site's masthead (e.g. exhibits/site/edit) that shows up on the exhibit's landing page.

Should that be the real "Site masthead" and then another term used for the exhibit mastheads/thumbnails themselves?  Perhaps it is inconsequential given the fact that only super admins will theoretically see the site-level masthead page and should know the difference.  Thoughts?

**Comment by ggeisler** [comment]
@jkeck I see what you mean. I'm not sure we should change the wording from what we previously discussed, though, since it is accurate when you are editing a specific exhibit. We could come up with different wording for the exhibits landing page case, but as you point out, only super-admins are going to see this (and even then, likely very rarely; we're still using the same masthead we started with a couple of years ago) so I don't think it will likely cause confusion.

So I'd say you have a good point but I'd recommend not worrying about it.

**Comment by ggeisler** [comment]
@jkeck Thinking about your suggestion a bit more, we could use the term "Exhibit masthead" and "Exhibit thumbnail" instead of "Site ..." for the actual exhibits. I kind of like that now that you got me thinking about it. Though if we do that we'd want to also change the titles of the tabs on the Appearance page to match, and then anywhere else we use "site" (not instances of "exhibit site" just "site" without "exhibit"), such as on the Dashboard page, where we'd change the first section heading to "Recent Exhibit Building Activity."

But maybe this should be a separate, low-priority ticket, since as discussed above we don't see a real problem with current situation?

**Comment by jkeck** [comment]
If it seems like it'll get tricky to get all the keys right, I'll create a new ticket for that.  Hopefully it won't be too much trouble.
